### PR TITLE
feat: base styles

### LINKED ISSUE #5: Global/Base Styling
- [x] fonts
- [x] colors
- [x] text sizes
- [x] margins/paddings
- [x] reusable svg images
- [ ] etc.

### PR DESCRIPTION
**Pull request by dicagno**
For a clear view on the implemented styles, reviewers might want to have a look at https://github.com/hlxsites/danaher-ls-aem/tree/3474334e4d1e9e895794e1d28835074704dd0b22/styles

Fix #5 

URL for testing:

- https://feat-global-base-styling--danaher-ls-aem--hlxsites.hlx.page/

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -249,3 +283,191 @@ main .section {
...
+  color: var(--color-gray-200);
+}
+
+.bg-danaherblue-600 {
+  background-color: var(--color-danaherblue-600);
+}
+
+.bg-danaherblue-900 {
+  background-color: var(--color-danaherblue-900);
+}
```
Comment (mhaack) [inline styles/styles.css:423]: Where these are needed? Lets not re-build tailwind style classes if we don't need these.

Comment (dicagno) [reply styles/styles.css:423]: Makes sense, we can directly link the custom properties where needed

File: styles/lazy-styles.css [hunk 2]
```diff
@@ -1 +1,5 @@
-/* add global styles that can be loaded post LCP here */
\ No newline at end of file
+/* add global styles that can be loaded post LCP here */
+
+/* Fort */
+@import url("https://use.typekit.net/oip4drb.css");
+@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
```
Comment (mhaack) [inline styles/lazy-styles.css:5]: Lets add the fonts directly to the projects, also make sure we have matching fallback fonts.

File: blocks/cards/cards.css
Comment (mhaack) [inline blocks/cards/cards.css:1]: This should not be here as not really part of global styles. Lets do this via #10. Anyway, lets keep it as a base version for the cards block.

File: styles/styles.css [hunk 3]
```diff
@@ -10,18 +10,47 @@
...
+  --shadow: 0 10px 15px -3px rgba(0, 0, 0, 0.1), 0 4px 6px -4px rgba(0, 0, 0, 0.1);
+  --shadow-colored: 0 10px 15px -3px var(--shadow-color), 0 4px 6px -4px var(--shadow-color);
+  --box-shadow-lg: var(--ring-offset-shadow, 0 0 #0000), var(--ring-shadow, 0 0 #0000), var(--shadow);
+
 
   /* fonts */
-  --body-font-family: roboto, roboto-fallback;
-  --heading-font-family: var(--body-font-family);
+  --body-font-family: inter, roboto-fallback;
+  --heading-font-family: fort, inter, roboto-fallback;
```
Comment (mhaack) [inline styles/styles.css:53]: Make sure we have the correct fallback fonts.

File: styles/styles.css [hunk 4]
```diff
@@ -249,3 +284,118 @@ main .section {
...
+
+/* various */
+.shadow-lg,
+.shadow-md {
+  box-shadow: var(--box-shadow-lg);
+}
+
+.rounded-full {
+  border-radius: 9999px;
+}
```
Comment (mhaack) [inline styles/styles.css:401]: Please remove, this can be added via individual block classes.

**Review by mhaack — CHANGES_REQUESTED** [review]

File: blocks/cards/cards.css
Comment (dicagno) [reply blocks/cards/cards.css:1]: Yeah, that's exactly why i implemented some basics as part of this task, to get a basic styling for cards as well.

File: styles/lazy-styles.css (hunk 2 above)
Comment (dicagno) [reply styles/lazy-styles.css:5]: For Inter, this should be possible; for Fort, this is a licensed font available in Adobe Fonts, so we have to use the hosted TypeKit css

File: styles/styles.css (hunk 3 above)
Comment (dicagno) [reply styles/styles.css:53]: Yes, will check this, thanks

File: styles/styles.css (hunk 4 above)
Comment (dicagno) [reply styles/styles.css:401]: Part of the task was about defining font sizes. Do you confirm these are not needed anymore in global styles?

File: styles/lazy-styles.css (hunk 2 above)
Comment (mhaack) [reply styles/lazy-styles.css:5]: Hm think you are right.

File: styles/fonts.css [hunk 5]
```diff
@@ -1,17 +1,27 @@
...
+  src: url('../fonts/roboto-bold.woff2') format('woff2');
+  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+}
+
+@font-face {
+  font-family: roboto;
+  font-style: normal;
+  font-weight: 400;
+  font-display: swap;
+  src: url('../fonts/roboto-regular.woff2') format('woff2');
```
Comment (mhaack) [inline styles/fonts.css:17]: This font is still missing  in the fonts folder

File: styles/fonts.css [hunk 6]
```diff
@@ -1,17 +1,27 @@
...
-    font-display: swap;
-    src: url('../fonts/roboto-regular.woff2') format('woff2');
-    unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
-  }
\ No newline at end of file
+  font-family: roboto;
+  font-style: normal;
+  font-weight: 700;
+  font-display: swap;
+  src: url('../fonts/roboto-bold.woff2') format('woff2');
```
Comment (mhaack) [inline styles/fonts.css:8]: This font is still missing  in the fonts folder

File: styles/fonts.css (hunk 5 above)
Comment (dicagno) [reply styles/fonts.css:17]: All the fonts are included in https://github.com/hlxsites/danaher-ls-aem/tree/4d96a8bc11690c4bd6a2c4caa192dd0cba6cfb89/fonts

Are you referring to the font files?

Comment (mhaack) [reply styles/fonts.css:17]: 👍

File: styles/fonts.css [hunk 7]
```diff
@@ -1,17 +1,27 @@
...
+  src: url('../fonts/roboto-regular.woff2') format('woff2');
+  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+}
+
+@font-face {
+  font-family: Inter;
+  font-style: normal;
+  font-weight: 400;
+  font-display: swap;
+  src: url('../fonts/inter-regular.ttf') format('truetype');
```
Comment (mhaack) [inline styles/fonts.css:26]: Why using older ttf font here, the original site loads https://fonts.gstatic.com/s/inter/v12/UcC73FwrK3iLTeHuS_fvQtMwCp50KnMa1ZL7W0Q5nw.woff2 for me?

Comment (dicagno) [reply styles/fonts.css:26]: If you try to download the official font pack from Google fonts, it only provides TTFs, that's why I used this format.
Aside from compression benefits, Is it a major concern? Just asking to know, more in general

**Comment by mhaack** [comment]
One more aspect: I see original uses tailwind "max-w-7xl" as max with for all major elements on the site, with "mx-auto" to keep it centered. We have it centered already but should go with "max-width: 80rem;" as well for our divs inside the sections

File: styles/fonts.css (hunk 7 above)
Comment (mhaack) [reply styles/fonts.css:26]: Interesting as woff2 is provided is used directly from Google. Let’s then as the Danaher colleagues on Slack.

Comment (dicagno) [reply styles/fonts.css:26]: yeah that's true, it's unusual. let's ask them then :)

**Review by mhaack — APPROVED** [review]

**Pull request merged**